test/tools/jsontests/BlockChainTestsBoost.cpp

```diff
@@ -173,4 +173,36 @@ BOOST_AUTO_TEST_CASE(userDefinedFile)
 	dev::test::userDefinedTest(dev::test::doBlockchainTests);
 }
 BOOST_AUTO_TEST_SUITE_END()
+
+//BlockChainTests
+BOOST_AUTO_TEST_SUITE(BlockChainTestsRandom)
+
+BOOST_AUTO_TEST_CASE(bcRandom)
```

Comment:
Do I understand the general idea correctly - this is a special-case code which is executed for RandomTests dir instead of the regular code in `doBlockchainTests` ?
Then perhaps it should be also in BlockChainTests.cpp, not here?